### PR TITLE
Added comments for methods, constructors and attributes

### PR DESCRIPTION
**Pull request by Kaspermn33**

### CODE REVIEW

File: src/worldofzuul/Room.java [hunk 1]
```diff
@@ -11,22 +11,48 @@
  */
 public class Room {
 
+    /**
+     * The String "description" is used when describing the room generated with the constructor
+     * The HashMap is a list of existing rooms
+     */
     private String description;
     private HashMap<String, Room> exits;
```
Comment (quartercastle) [inline src/worldofzuul/Room.java]: HashMap delen skal flyttes ned over denne attribute.

Comment (Kaspermn33) [reply src/worldofzuul/Room.java]: Fixed in 92bf61c54f269f9d29829785be5f57a245188487

File: src/worldofzuul/Room.java [hunk 2]
```diff
@@ -11,22 +11,48 @@
...
     public Room(String description) {
         this.description = description;
         exits = new HashMap<String, Room>();
     }
-
+    
+    /**
+     * The method sets the exits for the current room
+     * @param direction 
+     * Direction is the direction of the other rooms
```
Comment (quartercastle) [inline src/worldofzuul/Room.java]: Skal flyttes en linje op så.
```
@param direction Direction is the direction of the other rooms
```

Comment (Kaspermn33) [reply src/worldofzuul/Room.java]: Fixed in 20f03989dec3f91f64cdacc607da09352c372dcf

**Review by quartercastle — APPROVED** [review]
Ser godt ud 👍

**Pull request merged**